packages/_shared/package.json

```
@@ -47,7 +47,7 @@
   },
   "peerDependencies": {
     "@contentful/app-sdk": "^4.29.0",
-    "react": ">=16.8.0"
+    "react": ">=17.0.2"
```

```suggestion
    "react": "16 | 17 | 18"
```

otherwise, we would drop react 16 support ;)

---

Thanks, good spot!